src/modules/docker_context.rs

```diff
@@ -72,7 +72,11 @@ pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
                 _ => None,
             })
             .map(|variable| match variable {
-                "context" => Some(Ok(ctx.as_str())),
+                "context" => Some(Ok(if ctx.contains("unix://") {
+                    ""
+                } else {
+                    ctx.as_str()
+                })),
```

Comment:
I would prefer handling it like this:
```suggestion
                "context" if !ctx.starts_with("unix://") => ctx.as_str(),
```

Reply:
On the other hand, it might make sense to avoid displaying the module at all and returning `None` like when the value is `default`.

Reply:
Thank you for reviewing the PR 😄 . I've updated the codebase accordingly. Let me know if you have any more suggestions!

Reply:
I thought a bit about this, and please do move the `ctx.starts_with("unix://")` above and combine it with the `ctx == "default"` check.

Reply:
Sure @davidkna,
But, if we merge the check for the "default" context with the current condition, the `via 🐳` text will be omitted from our prompt when a Docker context is present. Is this an acceptable outcome?

Reply:
Yes.

Reply:
@davidkna  Refactored! 
I've squashed the irrelevant commits and retained only the first two for clarity. This ensures that if we need to revert in the future, the necessary changes are readily accessible. Thank you for reviewing the PR!